packages/core/core/src/Parcel.js

```diff
@@ -128,10 +129,14 @@ export default class Parcel {
       this.reporterRunner.report({
         type: 'buildSuccess',
         assetGraph,
+        changedAssets: new Map(this.assetGraphBuilder.changedAssets),
         bundleGraph,
         buildTime: Date.now() - startTime
       });
 
+      // TODO: Probably get a cleaner way to reset the map?
+      this.assetGraphBuilder.changedAssets = new Map();
```

Comment:
AssetGraphBuilder could probably reset the Map at the start of the `build` method?

Reply:
Yeah that would work, will change it